scripts/gaia_cat.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python -u
+
+"""
+Gaia catalog generator for roman simulator.
+
+"""
+
+import argparse
+import astroquery
+from astroquery.gaia import Gaia
+from astropy.time import Time
+from romanisim import gaia
+import romanisim.parameters, romanisim.bandpass
+
+def main():
+    """Main function of error estimator"""
+
+    # Define the parser
+    parser = argparse.ArgumentParser(description='Command and options for estimating fit uncertainties')
+
+    # Command line Argument
+    parser.add_argument('radecrad',
+                        type=float,
+                        metavar='178.389741 -2.678655, 1.0',
```

Comment:
Just because I don't know---is the python argument parser doing something "smart" like deleting that comma between ra/dec and radius?

Reply:
Just a typo. Fixing.